src/containers/Tenant/utils/paneVisibilityToggleHelpers.tsx

```diff
@@ -86,6 +86,7 @@ export function PaneVisibilityToggleButtons({
         <React.Fragment>
             <ActionTooltip title="Collapse">
                 <Button
+                    title="Collapse"
```

Comment:
Lets not use ActionTooltip together with native tooltip:
<img width="145" alt="Screenshot 2025-01-11 at 11 34 11" src="https://github.com/user-attachments/assets/baedd2bc-95e3-4728-8543-71d093c25523" />

Reply:
added type subclass to buttons